libdevcore/Log.cpp

```diff
@@ -43,7 +43,7 @@ void dev::simpleDebugOut(std::string const& _s, char const*)
 	static Mutex s_lock;
 	Guard l(s_lock);
 
-	cerr << _s << endl << flush;
+	cout << _s << endl << flush;
```

Comment:
I don't think logs should be going to std err by default, only program errors, no matter what log/debug level is being used. The most obvious effect of that is on the waterfall page, where it previously only got an `eth.err` logfile if there were actually errors, now we get both `eth.log` and `eth.err` no matter what, so it's impossible to tell whether there really was an error or not.